server/src/test/java/org/elasticsearch/cluster/metadata/MetadataIndexAliasesServiceTests.java

```diff
@@ -95,10 +95,15 @@ public void testAddAndRemove() {
 
         // Now just remove on its own
         before = after;
-        after = service.applyAliasActions(before, singletonList(new AliasAction.Remove(index, "test_2")));
+        after = service.applyAliasActions(before, singletonList(new AliasAction.Remove(index, "test_2", randomBoolean())));
         assertNull(after.metadata().getIndicesLookup().get("test"));
         assertNull(after.metadata().getIndicesLookup().get("test_2"));
         assertAliasesVersionIncreased(index, before, after);
+
+        final ClusterState finalCS = after;
```

Comment:
I would prefer if the `must_exist` option was tested in its own test method, so that it is clear that this is feature with different behaviour.

Reply:
I've pushed 4917987